Misc/sbom.spdx.json

```diff
@@ -1700,7 +1700,7 @@
       "checksums": [
         {
           "algorithm": "SHA256",
-          "checksumValue": "7ccf472345f20d35bdc9d1841ff5f313260c2c33fe417f48c30ac46cccabf5be"
+          "checksumValue": "5052d7889c1f9d05224cd41741acb7c5d6fa735ab34e339624a614eaaa7e7d76"
```

Comment:
Noting that this changed because the old one is for 23.2.1 (https://pypi.org/pypi/pip/23.2.1/json) and the new one is for 23.3.2 (https://pypi.org/pypi/pip/23.3.2/json), and the script is now calculating the value _and_ checking it matches the PyPI one 👍